pkg/ingester/client/circuitbreaker_test.go

```diff
@@ -78,7 +79,7 @@ func TestNewCircuitBreaker(t *testing.T) {
 
 	// Failed request that should put the circuit breaker into "open"
 	err = breaker(context.Background(), "/cortex.Ingester/Push", "", "", &conn, failure)
-	s, ok := status.FromError(err)
+	s, ok := grpcutil.ErrorToStatus(err)
```

Comment:
[nit]
Another possibility would be:

```suggestion
	code := grpcutil.ErrorToStatusCode(err)
```

and then to check 
```
require.Equal(t, codes.Unavailable, code)
```